common/pom.xml

```diff
@@ -165,6 +165,22 @@
                     <groupId>com.sun</groupId>
                     <artifactId>tools</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>servlet-api-2.5</artifactId>
```

Comment:
Well,  this and others that you mention so far are from the patch I got from Srini. While I made some changes and also moved around the actual jetty dependencies, I completely ignored the exclusions. Artifact id servlet-api-2.5 artifact-id doesnot look right. Thanks for pointing out, will send a patch.